pkg/graphql/jwt.graphql

```diff
@@ -0,0 +1,34 @@
+# DID specific schema
+
+extend type Query {
+    # Find edges
+    # returns an array of Edges
+    findEdges(fromDID: String!): [Edge!]!
```

Comment:
This will eventually need pagination, so I am concerned that we'll eventually either break this API or add a new method and deprecate this one. I guess we should just break the API since it is only us using this for now.

Reply:
Also might be more flexible for `findEdges` to accept an object of filters, and perhaps we only support `fromDID` - this gives us a more upgradable API